src/pytkdocs/loader.py

```diff
@@ -508,6 +509,7 @@ def get_class_documentation(self, node: ObjectNode, select_members=None) -> Clas
         for attr_name, properties, add_method in (
             ("__fields__", ["pydantic-model"], self.get_pydantic_field_documentation),
             ("_declared_fields", ["marshmallow-model"], self.get_marshmallow_field_documentation),
+            ("_meta.fields", ["django-model"], self.get_django_field_documentation),
```

Comment:
I think Django models were previously detected as Marshmallow ones. Do your changes fix that, and we don't need to put `_meta.fields` above `_declared_fields`?

Reply:
> I think Django models were previously detected as Marshmallow ones.

Really? I tried to use mkdocstrings for some Django projects and I haven't noticed that.
Django models do not have a `_declared_fields` attribute.

Reply:
Ah, you're right, it was for Django Rest Framework: https://github.com/mkdocstrings/mkdocstrings/issues/141